src/core/techs.jl

```diff
@@ -140,7 +147,7 @@ function Techs(s::Scenario)
     end
 
     if "Wind" in all_techs
-        append!(techs_no_turndown, ["Wind"])
+        append!(techs_no_turndown, ["Wind"]) # TODO: can this get moved to the if s.wind.max_kw > 0 statement?
```

Comment:
regarding the TODO: yes it can if you want to move it

Reply:
moved in https://github.com/NREL/REopt.jl/pull/75/commits/67e753736b25ac4f9549962b1dc2b53178821e63